.github/dependabot.yml

```diff
@@ -0,0 +1,47 @@
+version: 2
+updates:
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+      day: "sunday"
+      time: "02:00"
+    open-pull-requests-limit: 2
+    groups:
+      npm-minor-and-patch:
+        applies-to: version-updates
+        update-types:
+        - "patch"
+        - "minor"
+      npm-major:
+        applies-to: version-updates
+        update-types:
+        - "major"
```

Comment:
Can we remove major updates? These are typically significantly breaking and require manual intervention on most occasions so I prefer to evaluate them manually when releasing a new version.

Reply:
@Shaptic Done. Sorry, just came back to this.